{{ cookiecutter.repo_name }}/src/data/make_dataset.py

```diff
@@ -1,30 +1,47 @@
-# -*- coding: utf-8 -*-
-import click
-import logging
+import os
+import io
+import requests
+import pandas as pd
 from pathlib import Path
-from dotenv import find_dotenv, load_dotenv
```

Comment:
We'd prefer not to remove this (and the usage below) as it shows how to use a `.env` safely for credentials.